src/magic.h

```diff
@@ -177,7 +193,7 @@ class spell_type
         std::vector<fake_spell> additional_spells;
 
         // if the spell has a field name defined, this is where it is
-        cata::optional<field_type_id> field;
+        cata::optional<field_type_id> field = cata::nullopt;
```

Comment:
This change is redundant; it was already initializing to an empty optional.  But if you want this for extra clarity, I suppose it's OK.